internal/service-manager/client_test.go

```diff
@@ -128,6 +161,97 @@ func (h *fakeSMHandler) getServiceOfferings(w http.ResponseWriter, r *http.Reque
 	}
 }
 
+func (h *fakeSMHandler) getServiceOffering(w http.ResponseWriter, r *http.Request) {
+	soID := r.PathValue("serviceOfferingID")
+	if len(soID) == 0 {
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	data, err := json.Marshal(h.serviceOfferings)
+	if err != nil {
+		log.Println("error while marshalling service offerings data: %w", err)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	var sos types.ServiceOfferings
```

Comment:
Nit pick: You sometimes use `so` and sometimes `sos` for ServiceOfferings.

Reply:
I meant to use `so` for `types.ServiceOffering` and `sos` for `types.ServiceOfferings` which are two distinct structs. I will change it to something more distinguishable and meaningful.

Reply:
The thing is I am ok with short names like 'so' and 'sos' but in some places you use 'so' for (plural) service offerings. I expected the convention you mentioned in the reply. so for singular, sos for plural.

Reply:
See line 186 below.

Reply:
Ah, I see it now, thanks.

Reply:
Fixed typos.